arcgis-ios-sdk-samples/Search/Find address/README.md

```diff
@@ -1,19 +1,31 @@
 # Find address
 
-This sample demonstrates how to geocode an address and show it on the map view
+Find the location for an address.
 
+## Use case
 
-## How to use the sample
-
-You can either type in an address or choose an address from the list to show it on the map view. The address location is marked with a pin. You can tap on the pin to show the address in a callout.
+A user can input a raw address into your app's search bar and zoom to the address location.
 
+## How to use the sample
 
-![](image1.png)
+For simplicity, the sample comes loaded with a set of suggested addresses. Choose an address from the suggestions or submit your own address to show its location on the map in a callout. Tap the pin to display the address on the map.
 
+![Find address sample](find-address.png)
 
 ## How it works
 
-The sample uses the `geocode(withSearchText:parameters:completion:)` method on the `AGSLocatorTask` to geocode an address. It returns an array of `AGSGeocodeResult` objects in case of success. The first result is displayed on the map view using `AGSGraphicsOverlay`. The attributes returned in the `AGSGeocodeResult` object are used as `title` and `detail` fields for the callout.  To show the callout the sample uses the `show(for:tapLocation:animated:)` method on `AGSCallout`.
+1. Create an `AGSLocatorTask` using the URL to a locator service.
+2. Initialize `AGSGeocodeParameters` and specify the `AGSGeocodeParameters.resultAttributeNames`.
```

Comment:
This is better, but it still isn't quite accurate as there is such as thing as initializing a class (rather than an instance of a class). I suggest using "Create an `AGSGeocodeParameters` object…" or "Create an instance of `AGSGeocodeParameters`…".